### PR TITLE
feat(secureboot-certs): force default options when no install arguments are passed

### PR DESCRIPTION
**Pull request by beshleman**
To help reduce the amount of typing required for the common case, this
commit invokes all of the default and latest certificates if no args are
passed to the install command

### CODE REVIEW

**Review by stormi — APPROVED** [review]

**Comment by stormi** [comment]
Now an update to the doc PR is needed

**Comment by beshleman** [comment]
I am wary to merge this given the number of users who will break their VMs accidentally with the `latest` dbx.

Thoughts @stormi ?

**Comment by stormi** [comment]
> I am wary to merge this given the number of users who will break their VMs accidentally with the `latest` dbx.
> 
> Thoughts @stormi ?

Users won't break existing VMs when they first setup their pools, since secure boot won't be on for their existing VMs. They will simply discover, if they try to enable SB on a linux VM, what the painful and complex situation SB on linux currently is, due to the certificate revocations. Unless they've read our docs first, where we'll explain it.

And if they run the command later, in a way to update the certs on the pool, the existing VMs won't automatically start using the new dbx so they should still work.

So I'm for the merge.

**Comment by stormi** [comment]
What we could add to this PR is printing something like
```
No arguments provided to command install, default arguments will be used: 
- PK: default
- KEK: default
- db: default
- dbx: latest
```

File: scripts/secureboot-certs [hunk 1]
```diff
@@ -446,10 +446,17 @@ if __name__ == "__main__":
     install_parser = action_parsers.add_parser(
         Actions.INSTALL,
         help="Install UEFI certificates to every host in the pool.",
-        description="Install UEFI certificates to every host in the pool.",
+        description=(
+              "Install UEFI certificates to every host in the pool.\n\n"
+              "If no arguments are passed to '{} {}', then the default PK, KEK, "
+              "and db, and the latest dbx will be installed by default."
```
Comment (stormi) [inline scripts/secureboot-certs:452]: There's "default" twice in the sentence. Would it be better without the second occurrence?

Comment (beshleman) [reply scripts/secureboot-certs:452]: Yes, I think so.

**Pull request merged**